app/models/proceeding_type.rb

```diff
@@ -7,11 +7,15 @@ class ProceedingType < ApplicationRecord
 
   validates :code, presence: true
 
-  scope :with_ccms_code_starting, ->(_text) do
-    where(arel_table[:ccms_code].matches_regexp('^DA\d+$'))
-  end
-
   def self.populate
     ProceedingTypePopulator.call
   end
+
+  def default_substantive_scope_limitation
+    proceeding_type_scope_limitations.where(substantive_default: true).first.scope_limitation
```

Comment:
would this work?
```suggestion
    proceeding_type_scope_limitations.find_by(substantive_default: true).scope_limitation
```

Reply:
have delegated this out to the proceeding_types_scope_limtations table..

But is `where...first` preferable to  `find_by`

Reply:
I think nowadays both result in the same SQL call. Personally, I think `find_by` is more succinct; why use two methods when one will do.